--- a/tomcat6  2012-02-06 14:03:44.000000000 +0100
+++ b/tomcat6  2012-07-24 14:47:39.000000000 +0200
@@ -187,6 +187,7 @@
 # See how we were called.
 function start() {
     echo -n "Starting Tomcat ($CATALINA_BASE)"
+    sed -i 's/^#\(.*Xdebug\)/\1/' /etc/tomcat6/tomcat6.conf
     if [ -f "/var/lock/subsys/${NAME}" ] ; then
         if [ -f "/var/run/${NAME}.pid" ]; then
             read kpid < /var/run/${NAME}.pid
@@ -266,6 +267,7 @@

 function stop() {
     echo -n "Shutting down Tomcat ($CATALINA_BASE)"
+    sed -i 's/^\(J.*Xdebug\)/#\1/' /etc/tomcat6/tomcat6.conf
     if [ -f "/var/lock/subsys/${NAME}" ]; then
         $SU - $TOMCAT_USER -c "$TOMCAT_SCRIPT stop" >> $TOMCAT_LOG 2>&1
         RETVAL="$?"